### DISCUSSION TITLE
How to generate excel file and send to client with axum get handle ,

### DISCUSSION THREAD

**Discussion by toplinuxsir** [tokio-rs/axum#1434] [Q&A]
## Feature Request

How to generate excel file and send to client with axum get handle ,  Any example ? Thanks!

**Comment by davidpdrsn** [comment]
I have never done excel stuff from Rust. Not sure if there is a crate for it. You’re probably better off asking on the rust users forum about that.

To return it from an axum handler you need to return it as bytes from some kind. Probably Vec<u8> or maybe String. All that depends on the excel crate you use.
